x-pack/test/performance/services/performance.ts

```diff
@@ -164,6 +169,44 @@ export class PerformanceTestingService extends FtrService {
     return client;
   }
 
+  private telemetryTrackerCount = 0;
+
+  private trackTelemetryRequests(page: Page) {
+    const id = ++this.telemetryTrackerCount;
+
+    const requestFailure$ = Rx.fromEvent<Request>(page, 'requestfailed');
+    const requestSuccess$ = Rx.fromEvent<Request>(page, 'requestfinished');
+    const request$ = Rx.fromEvent<Request>(page, 'request').pipe(
+      Rx.takeUntil(
+        this.pageTeardown$.pipe(
+          Rx.first((p) => p === page),
+          Rx.delay(3000)
+          // If EBT client buffers:
+          // Rx.mergeMap(async () => {
+          //  await page.waitForFunction(() => {
+          //    // return window.kibana_ebt_client.buffer_size == 0
+          //  });
+          // })
```

Comment:
In the description you mention

> We are not sure if browser buffers the event requests to server, if that is the case we have to check if buffer [cleaned up](https://github.com/elastic/kibana/compare/main...suchcodemuchwow:fix-wait-for-journeys-to-finish-reqs?expand=1#diff-2fb328c51929720e143497f07d990b1763289793872dbeb9ce7e148839251ba0R187)

I don't think we have to do this, though we might want to down the road if this proves insufficient because of the way the buffer in the browser client works.

Reply:
Wouldn't it be hard to understand we are missing some events unless we really dig into logs?

Reply:
I expect if we're missing events we will see holes in our charts